aten/src/TH/THGenerateBFloat16Type.h

```diff
@@ -4,11 +4,13 @@
 
 #include <c10/util/BFloat16.h>
 #define scalar_t at::BFloat16
+#define accreal double
```

Comment:
@izdeby why is accreal double here?  Naively I'd expect the same accreal for float16, which is float.